admin-dev/themes/new-theme/scss/pages/product/products_catalog.scss

```diff
@@ -57,6 +57,10 @@
       font-weight: 400;
     }
 
+    #filter_column_id_product_div {
```

Comment:
@NeOMakinG from what I know `id` column is too big on every page, Categories too, I think we need something more generic?

Reply:
Ok, gonna think about it

Reply:
@kpodemski would you mind giving help on this? It seems that you're right, some id's are big too, but I've no way of selecting them globally as they doesn't have a specific class everywhere :/

I would expect something like a class named `grid-head-id` on every head related to the id column

Reply:
@NeOMakinG 

https://github.com/PrestaShop/PrestaShop/issues/22847

haha :) so now we have an use case, I'll handle this tomorrow 👍

Reply:
@NeOMakinG done, you can check if that's ok :)

Reply:
Ok, gonna look at it today :)

Reply:
Thanks!